src/components/home/Header.js

```diff
@@ -20,7 +20,7 @@ export default class Header extends Component {
       };
     }
     fetchData(){
-        alert('Refresh');
+        alert('Refresh/Get New Data');
```

Comment:
what this do?

Reply:
or rather why this change?

Reply:
its an alert box. I did that for me just to remember what is happening at that point.

Reply:
ok i see... yeah i thoughtt he / did something fancier ;p